spec/models/sitemap_spec.rb

```diff
@@ -15,14 +17,14 @@
 
   describe 'validations' do
     context 'when validating url uniqueness' do
-      let!(:original) { described_class.create!(valid_attributes) }
+      before { described_class.create!(valid_attributes) }
 
       it 'rejects duplicate urls' do
-        expect(described_class.new(valid_attributes)).to_not be_valid
+        expect(described_class.new(valid_attributes)).not_to be_valid
       end
 
       it 'is not case-sensitive' do
-        expect(described_class.create!(url: url.upcase)).to be_valid
+        expect(described_class.new({ url: url.upcase })).not_to be_valid
```

Comment:
For tidiness, we can omit the curly brackets:
```ruby
expect(described_class.new(url: url.upcase)).not_to be_valid
```

Reply:
Change made.